auth-web/src/views/auth/HomeViewDev.vue

```diff
@@ -69,10 +69,13 @@
         <v-container>
           <h2>How does it work?</h2>
           <InfoStepper />
-          <RouterView
-            :userProfile="userProfile"
-            @account-dialog="accountDialog = true"
-            @manage-businesses="goToManageBusinesses($event)"/>
+          <transition name="slide-x-transition">
```

Comment:
Nice 👍

Reply:
Yea! Kinda cool, applying a transition to a routerview. Vuetify is pretty solid :)